src/Support/Helpers/Report.php

```diff
@@ -10,14 +10,18 @@
      */
     function bugsnag_report(\Exception $exception)
     {
-        // Retrieve Bugsnag instance
-        $bugsnag = app('nodes.bugsnag');
-
-        // Report exception to Bugsnag
-        if ($exception instanceof \Nodes\Exceptions\Exception) {
-            $bugsnag->notifyException($exception, $exception->getMeta(), $exception->getSeverity());
-        } else {
-            $bugsnag->notifyException($exception, null, 'error');
+        try {
+            // Retrieve Bugsnag instance
+            $bugsnag = app('nodes.bugsnag');
+    
+            // Report exception to Bugsnag
+            if ($exception instanceof \Nodes\Exceptions\Exception) {
+                $bugsnag->notifyException($exception, $exception->getMeta(), $exception->getSeverity());
+            } else {
+                $bugsnag->notifyException($exception, null, 'error');
+            }   
+        } catch(\Exception $e) {
+            // Fail silent
```

Comment:
Fair point. But why is it failing locally? If it's cause of the Bugsnag API key missing you can add it in config/nodes/Bugsnag.php or in the .env with BUGSNAG_API.

Really think is try-catch is redundent.